src/components/layout/AddMovie.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export const AddMovie = () => {
+  return (
+    <div className='addMovie_container '>
+      <div className='addMovie_tag '>
+         movies<label style={{fontWeight:'bold'}}>DB</label>
```

Comment:
Minor: don't break HTML semantic, there is no reason to use `label` here. This might look harmless but actually messes up experience with assistive technologies such as screen readers.